res/TensorFlowLiteRecipes/Inf_StridedSlice_000/test.recipe

```diff
@@ -0,0 +1,47 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 0 dim: 0 dim: 5 }
+  shape_signature { dim: 1 dim: -1 dim: -1 dim: 5 }
+}
+operand {
+  name: "begin"
+  type: INT32
+  shape { dim: 4 }
+  filler { tag: "explicit" arg: "0" arg: "0" arg: "0" arg: "0" }
+}
+operand {
+  name: "end"
+  type: INT32
+  shape { dim: 4 }
+  filler { tag: "explicit" arg: "1" arg: "8" arg: "3" arg: "5" }
+}
+operand {
+  name: "strides"
+  type: INT32
+  shape { dim: 4 }
+  filler { tag: "explicit" arg: "1" arg: "1" arg: "1" arg: "1" }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 0 dim: 0 dim: 5 }
+  shape_signature { dim: 1 dim: -1 dim: -1 dim: 5 }
```

Comment:
I'll leave this comment for the record.

Theoretically, in this case, it seems that the shape of the `output tenser` can be determined by the `begin`, `end`, and `stride` values.

I think this can be solved by updating the `tflite` code part together later.

I will create a separate issue related to this.